lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/RootContextLauncher.cs

```diff
@@ -56,8 +56,17 @@ internal ContextRuntime GetRootContext()
         {
             if (_rootContext == null)
             {
-                // TODO[JIRA REEF-1167]: Remove use of this constructor.
-                _rootContext = new ContextRuntime(Id, _rootServiceInjector, _rootContextConfiguration);
+                // TODO[JIRA REEF-1167]: Remove use of deprecated ContextRuntime constructor and deprecatedContextConfiguration
+                var deprecatedContextConfiguration = _rootContextConfiguration as ContextConfiguration;
```

Comment:
_rootContextConfig comes from constructor, which is from evaluatorSettings.RootContextConfig that is IConfiguration. Do we have scenario where _rootContextConfig is ContextConfiguration?